src/components/NcDateTimePicker/NcDateTimePicker.vue

```diff
@@ -207,26 +217,28 @@ const props = withDefaults(defineProps<{
 	 */
 	confirm?: boolean
 
-	/**
-	 * Default increment step for minutes in the time picker.
-	 * @default 10
-	 */
-	minuteStep?: number
-
 	/**
 	 * Preview format for the picker input field.
+	 * Can either be a string of Unicode tokens or a function that takes a date,
```

Comment:
I would specify here 
```suggestion
	 * Can either be a string of Unicode tokens or a function that takes a Date object
```

Reply:
makes sense, thank you!